<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecryptionWidget</class>
 <widget class="QWidget" name="DecryptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DecryptionWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="keyLedr">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="showHideBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="BCipher.qrc">
         <normaloff>:/Resources/hide.svg</normaloff>:/Resources/hide.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="intputBrsBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="BCipher.qrc">
         <normaloff>:/Resources/browser.svg</normaloff>:/Resources/browser.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileNameLedr"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="keyLbl">
       <property name="text">
        <string>Khóa giải mã:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="fileNameLbl">
       <property name="text">
        <string>File cần giải mã:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="originLbl">
       <property name="text">
        <string>File giải mã:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="originLedr">
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="originBrsBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="BCipher.qrc">
         <normaloff>:/Resources/browser.svg</normaloff>:/Resources/browser.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>178</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="decryptionBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Giải mã</string>
       </property>
       <property name="icon">
        <iconset resource="BCipher.qrc">
         <normaloff>:/Resources/unlock.svg</normaloff>:/Resources/unlock.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BCipher.qrc"/>
 </resources>
 <connections/>
</ui>
